Research/MigrationNotes_v3.3.md

```diff
@@ -0,0 +1,30 @@
+#  Migration Steps -> v3.3
+
+Default task decoding using `open func decodeTask(from decoder: Decoder) throws -> RSDTask`  and
+`RSDTaskObject` are deprecated. Instead, register tasks either by registering your own `GenericSerializer` with 
+your factory or by adding your custom tasks to the task serializer. Transformers are all still handled the same way.
+
+
+For example:
+
+```
+class MyFactory : RSDFactory {
+    override public init() {
+        super.init()
+        self.stepSerializer.add(MyCustomStep(identifier: "foo"))
```

Comment:
Tbh I'm more confused than illuminated by this example. You define a MyCustomTask that isn't described in the text or referenced anywhere in the example, and the example references MyCustomStep and MyStepNavigator, neither of which are defined or described hereabouts. I'm guessing there's an assumption here that since these are migration notes that the reader is migrating something that has all these missing pieces so they already know what they are, but it might be better to make that explicit?

Reply:
Typo. Fixed.